src/components/primitives/Links/styles.tsx

```diff
@@ -0,0 +1,25 @@
+import styled from 'styled-components';
+import { Box } from '../Layout';
+
+export const StyledSegmentLink = styled.a`
+  color: ${({ theme }) => theme.colors.grey};
+  margin-right: 0.5rem;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  text-decoration: none;
+  overflow-wrap: break-word;
+  &:hover {
+    cursor: pointer;
+    color: ${({ theme }) => theme.colors.text};
+  }
+`;
+
+export const FlexContainer = styled.div`
+  display: flex;
+`;

```

Comment:
```suggestion
export const FlexContainer = styled.div`
  display: flex;
`;

```

Removing this (see comment above)

Reply:
great to know